### PR TITLE
Feature/add new task

### PR DESCRIPTION
**Pull request by raitasu**
add adding feature

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,100 +1,123 @@
...
-        console.log("completed");
-        break;
-      }
-      default: {
-        setTasks(initTasks);
-        console.log("all or smth");
-      }
+    const handleClickDeleteTask = (id: string) => {
+        const newTaskList = tasks.filter((t) => {
+            // if (t.id !== id) {
```
Comment (Vladqqqqoo) [inline src/App.tsx:40]: please, remove commented code

Comment (raitasu) [reply src/App.tsx:40]: will remove later

Comment (Vladqqqqoo) [reply src/App.tsx:40]: ok

**Comment by Vladqqqqoo** [comment]
approve

**Pull request merged**